lib_comfyui/custom_extension_injector.py

```diff
@@ -2,56 +2,74 @@
 import inspect
 import ast
 import textwrap
+from lib_comfyui.find_extensions import get_extension_paths_to_load
+
 
 webui_custom_nodes_path = os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), 'comfyui_custom_nodes')
 webui_custom_scripts_path = os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), 'comfyui_custom_scripts')
+webui_extensions_path = os.path.dirname(os.path.dirname(os.path.dirname(os.path.realpath(__file__))))

```

Comment:
Use `shared.extensions_dir` instead

Reply:
And do we still need the `webui_custom_*_path` variables? IIUC it should be covered by the extension loader below